tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -49,8 +49,8 @@ const (
 	ruleBundlePrefix              = "pd/api/v1/config/placement-rule"
 	pdServerPrefix                = "pd/api/v1/config/pd-server"
 	serviceMiddlewareConfigPrefix = "pd/api/v1/service-middleware/config"
-	// flagFromAPIServer has no influence for pd mode, but it is useful for us to debug in api mode.
-	flagFromAPIServer = "from_api_server"
+	// flagFromPDService has no influence for pd mode, but it is useful for us to debug in pd service mode.
+	flagFromPDService = "from_api_server"
```

Comment:
Do we need to change "from_api_server"?

Reply:
I didn't change it before because I'm afraid there is a compatibility issue. Since it is not GA now, I'm going to change it.